torchx/runner/events/__init__.py

```diff
@@ -59,6 +59,15 @@ def _get_or_create_logger(destination: str = "null") -> logging.Logger:
 def record(event: TorchxEvent, destination: str = "null") -> None:
     _get_or_create_logger(destination).info(event.serialize())
 
+    if destination != "console":
```

Comment:
would it make sense to have a console event handler, so we don't need this check?